cmd/ci-secret-bootstrap/main.go

```diff
@@ -741,23 +741,59 @@ func getUnusedBWItems(config secretbootstrap.Config, client secrets.ReadOnlyClie
 	return utilerrors.NewAggregate(errs)
 }
 
+func validateBWItems(client secrets.ReadOnlyClient, secretConfigs []secretbootstrap.SecretConfig) error {
+	var errs []error
+
+	for _, config := range secretConfigs {
+		for _, item := range config.From {
+
+			if item.DockerConfigJSONData != nil {
+				for _, data := range item.DockerConfigJSONData {
+					if !client.HasItem(data.BWItem) {
+						errs = append(errs, fmt.Errorf("item %s doesn't exist", data.BWItem))
+						continue
```

Comment:
This `continue` does not make sense - the next check is for an attachment to this item, so if the item does not exist there's no point in checking for the attachment.

Reply:
:facepalm:  Let me fix this

Reply:
updated